jenkins_pipelines/environments/common/pipeline-mu-aws.groovy

```diff
@@ -37,52 +42,100 @@ def run(params) {
-                    stage("Create empty AWS mirror") {
-                        env.aws_configuration = "REGION = \"${params.aws_region}\"\n" +
-                                "AVAILABILITY_ZONE = \"${params.aws_availability_zone}\"\n" +
-                                "NAME_PREFIX = \"${env.JOB_NAME}-\"\n" +
-                                "ALLOWED_IPS = [ \n"
-
-                        ALLOWED_IPS.each { ip ->
-                            env.aws_configuration = aws_configuration + "    \"${ip}\",\n"
+                        // Deregister all BYOS images
+                        AMI_LIST.each { ami ->
+                            if (ami) {
+                                sh(script: "${awscli} ec2 deregister-image --image-id ${ami} --region ${params.aws_region}")
+                            }
                         }
+                        // Delete all BYOS snapshot
+                        SNAPSHOT_LIST.each { snapshot ->
+                            if (snapshot) {
+                                sh(script: "${awscli} ec2 delete-snapshot --snapshot-id ${snapshot} --region ${params.aws_region}")
+                            }
+                        }
+                    }
 
-                        env.aws_configuration = aws_configuration + "]\n"
-                        writeFile file: "${aws_mirror_dir}/terraform.tfvars", text: aws_configuration, encoding: "UTF-8"
 
-                        // Deploy empty AWS mirror
-                        sh "set +x; source /home/jenkins/.credentials set -x; source /home/jenkins/.aws set -x;source /home/jenkins/.registration set -x; export TF_VAR_CUCUMBER_GITREPO=${params.cucumber_gitrepo}; export TF_VAR_CUCUMBER_BRANCH=${params.cucumber_ref}; export TERRAFORM=${params.terraform_bin}; export TERRAFORM_PLUGINS=${params.terraform_bin_plugins}; ./terracumber-cli ${aws_mirror_params} --logfile ${resultdirbuild}/sumaform-aws.log ${TERRAFORM_INIT} --taint '.*(domain|main_disk).*' --runstep provision --sumaform-backend aws"
-                        deployed_aws = true
+                    stage('Download last ami image') {
+                        sh "rm -rf ${resultdir}/images"
+                        sh "mkdir -p ${resultdir}/images"
+                        server_image_name = sh(script: "curl https://dist.suse.de/ibs/SUSE:/SLE-15-SP4:/Update:/Products:/Manager43/images/ | grep -oP '<a href=\".+?\">\\K.+?(?=<)' | grep 'SUSE-Manager-Server-BYOS.*raw.xz\$'",
+                                returnStdout: true).trim()
+                        proxy_image_name = sh(script: "curl https://dist.suse.de/ibs/SUSE:/SLE-15-SP4:/Update:/Products:/Manager43/images/ | grep -oP '<a href=\".+?\">\\K.+?(?=<)' | grep 'SUSE-Manager-Proxy-BYOS.*raw.xz\$'",
+                                returnStdout: true).trim()
+                        sh(script: "cd ${resultdir}/images; wget https://dist.suse.de/ibs/SUSE:/SLE-15-SP4:/Update:/Products:/Manager43/images/${server_image_name}")
```

Comment:
That could be a constant `https://dist.suse.de/ibs/SUSE:/SLE-15-SP4:/Update:/Products:/Manager43/images/`